static/swagger-specs/unified-tags.yaml

```diff
@@ -0,0 +1,983 @@
+          example: 1705624523000
+        createdBy:
+          type: string
+          description: "The ID of the user who created the tag."
+          example: "{USER_ID}"
+        id:
+          type: string
+          description: "The ID of the tag."
+          example: "8af14b1e-f267-44ad-b94c-9ac70274e3d5"
+        modifiedAt:
+          type: string
+          description: "The timestamp that the tag was last updated."
+          example: "1705624523000"
+        modifiedBy:
+          type: string
+          description: "The ID of the user who last updated the tag."
+          example: "{USER_ID}"
+        name:
+          type: string
+          description: "The name of the tag."
+          example: "sample-tag"
+        org:
+          type: string
+          description: "The organization ID of the tag."
+          example: "{ORG_ID}"
+        tagCategoryId:
+          type: string
+          description: "The ID of the tag category that the tag belongs to."
+          example: "e2b7c656-067b-4413-a366-adde0401df50"
+        tagCategoryName:
+          type: string
+          description: "The name of the tag category that the tag belongs to."
+          example: "Test Category"
+    tagsList:
+      type: object
+      properties: 
+        "_page":
+          type: object
+          properties: 
+            count:
+              type: integer
+              description: "The number of tags that match the query given."
+              example: 166
+            limit:
+              type: integer
+              description: "The maximum number of tags that appear per page."
+              example: 10
+            next:
+              type: string
+              description: "The string that lets you visit the next page of results."
+              example: eyJjb21wb3NpdGVUb2tlbiI6IntcInRva2VuXCI6XCIrUklEOn52a0owQUp3WDRrVko1d0FBQUFBQUFBPT0jUlQ6MiNUUkM6MjAjUlREOnVDTmQyWlAvWjV6TGdvUGVGR1JHQk1KNVExVmR6Mnc9I0lTVjoyI0lFTzo2NTU2NyNRQ0Y6OCNGUEM6QWdFQ0J3TG1BQ1NmQnNBQ0JBb0FBQVFBQ0FBQUNJQVlnQWVBRElBTmdBWEFFTUJCUUVBQUFBQkFRQkdBSElBR2dBQ0FENEFId0FKUkRFQUNBZ2dBUUJnQUVBQUlIb0FaZ0FDQUJNQUFRVUFBUUFCQVFScUFBc0FTQUFBRUxvQU9nQWFBQmNBQVlBQUFHSUlCUUFDQU1vQUlnQWlBQk1DQUFRQUFnZ0FnQUM2QURZQTNnQWlBR1lBQWdCZUFBY0FCZ0JlQUM4QURBQUlBQWdBQVFBQ0FBRUZBQVFFQUFBRWdBQ0FBSjRCR2dBeUFCSUFPZ0F5QU13QVNRQ0FBQUVBdGdCRUFBR0FkZ0FuQUFDZ0NBQUFBQ0lCQUFDSkFnQUJBRUFDQUg0QUhnQWFBQllBVUFFQUNCQUFFQUFRQUF4QUFzUnJBQUlFQUFBYkxoQklIQVBBQUhnUUVBTEVxQUE4RkNBQVFtcUVBd0FBTWd3Y09BSFdIa1FBZ0JGT0FTNEN4QVE0QVwiLFwicmFuZ2VcIjp7XCJtaW5cIjpcIlwiLFwibWF4XCI6XCJGRlwifX0iLCJvcmRlckJ5SXRlbXMiOlt7Iml0ZW0iOjE2OTQ0ODg2MDMwMDB9XSwicmlkIjoidmtKMEFKd1g0a1hHV2dFQUFBQUFBQT09IiwiaW5jbHVzaXZlIjp0cnVlfQ"
+            property:
+              type: array
+              items:
+                type: string
+                description: "The query that limits the tags retrieved."
+                example: "tagCategoryId=e2b7c656-067b-4413-a366-adde0401df50"
```

Comment:
For some reason, this description isn't displaying properly.
![Screen Shot 2024-06-28 at 12 29 27 PM](https://github.com/AdobeDocs/experience-platform-apis/assets/43826746/8b1e776d-556d-430b-b2be-2a0c6efb7e52)

Reply:
Could be a weird rendering issue. I'll test this out when I deploy on Stage.